pkg/ytconfig/canondata/TestGetHTTPProxyConfig/test.canondata

```diff
@@ -109,8 +109,10 @@
             ];
         };
         "oauth_cookie_authenticator"={
+            "create_user_if_not_exists"=%false;
```

Comment:
What is this canondata? Does it mean this is default behaviour if a user did not specify `DisableUserCreation`?

`create_user_if_not_exists` should be not set if user did not specify `DisableUserCreation`, and set to `!DisableUserCreation` otherwise

Reply:
no, i've added true in test oauth spec